Packages/MIES/json_functions.ipf

```diff
@@ -428,7 +428,7 @@ Function JSON_AddVariable(jsonID, jsonPath, value, [significance, ignoreErr])
 
 	ignoreErr = ParamIsDefault(ignoreErr) ? JSON_ZFLAG_DEFAULT : !!ignoreErr
 
-	if(trunc(value) == value)
+	if(trunc(value) == value && !numtype(value))
```

Comment:
Good find. Let's use the latest JSON XOP and procedure file now that this changes is upstream.